CMakeLists.txt

```diff
@@ -23,9 +23,7 @@ cm_workspace(crypto3)
 cm_setup_version(VERSION 0.3.0 PREFIX ${CMAKE_WORKSPACE_NAME})
 
 option(BUILD_SHARED_LIBS "Build shared library" FALSE) # TODO: it makes no sense for header-only lib, remove
-option(USE_STATIC_BOOST "Use static libraries for Boost" FALSE)
-option(BUILD_TESTS "Include tests to `all` target" FALSE) # used by CMTest module
-option(CMAKE_ENABLE_TESTS "Enable tests" FALSE) # used by CMTest module
+option(ENABLE_TESTS "Enable tests" TRUE) # used by CMTest module
```

Comment:
Have you checked it? Try to remove CMAKE_ENABLE_TESTS and add ENABLE_TESTS to nix